doc/articles/controls/GLCanvasElement.md

```diff
@@ -47,7 +47,7 @@ To learn more about using [Silk.NET](https://www.nuget.org/packages/Silk.NET.Ope
 
 ## Full example
 
-To see this in action, here's a complete sample that uses `GLCanvasElement` to draw a triangle. Note how you have to be careful with surrounding all the OpenGL-related logic in platform-specific guards. This is the case for both the [XAML](platform-specific-xaml) and the [code-behind](platform-specific-csharp).
+To see this in action, here's a complete sample that uses `GLCanvasElement` to draw a triangle. Note how you have to be careful with surrounding all the OpenGL-related logic in platform-specific guards. This is the case for both the [XAML](platform-specific-xaml) and the [code-behind](platform-specific-csharp). For complete C# projects, visit our [GLCanvasElement Samples in the Uno.Samples repository](https://aka.platform.uno/glcanvaselement-sample).
```

Comment:
@jeromelaban The link here should match the one in the feature announcement (just in case you change it).

Reply:
Thanks! It will be updated automatically.